kanidmd/score/src/lib.rs

```diff
@@ -484,6 +483,42 @@ pub fn verify_server_core(config: &Configuration) {
     // Now add IDM server verifications?
 }
 
+// TODO: should this go somewhere else
```

Comment:
I think it can stay with account recover here, but in the future I might split the core.rs into each function as a little bundle of joy.

Reply:
Yeah, the idea might be to move the JSON-related structs for messages out into the `kanidm` crate since that should limit duplication.